src/sdk/object_sdk.js

```diff
@@ -227,6 +252,8 @@ class ObjectSDK {
             this.requesting_account = await account_cache.get_with_cache({
                 bucketspace: this._get_bucketspace(),
                 access_key: token ? token.access_key : anonymous_access_key,
+                validation_callback: _validate_account,
+                valid_until: Date.now() + config.OBJECT_SDK_ACCOUNT_CACHE_EXPIRY_MS,
```

Comment:
if we always pass 10 minutes from now to validate_account and validate_account check `  if (time <= params.valid_until) return true;` wouldn't it always be valid?

Reply:
Thank you, I changed the implementation, I hope it is fine now.